src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/LabelRenderer.cs

```diff
@@ -267,11 +265,23 @@ void UpdateFont(TextBlock textBlock)
 			if (label == null || (label.IsDefault() && !_fontApplied))
 				return;
 
-#pragma warning disable 618
-			Font fontToApply = label.IsDefault() && _isInitiallyDefault ? Font.SystemFontOfSize(NamedSize.Medium) : label.Font;
-#pragma warning restore 618
+			if (label.IsDefault() && _isInitiallyDefault)
+			{
+				var font = Font.SystemFontOfSize(NamedSize.Medium);
```

Comment:
Where is this code coming from?

Reply:
That was in the previous PR... Not sure what it should be